web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralParagraphTranslatability.php

```
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\Tests\server_general\ExistingSite;
+
+use weitzman\DrupalTestTraits\ExistingSiteBase;
+
+/**
+ * Tests entity references.
+ */
+class ServerGeneralParagraphTranslatability extends ExistingSiteBase {
+
+  /**
+   * Checks all entity reference revision fields for Paragraphs are non-translatable.
+   */
+  public function testParagraphsTranslation() {
```

Another idea that might be better - can we throw an Exception if one tries to set the field incorrectly?

---

Good point :+1: Save ourselves from ourselves :)